### PR TITLE
Report changes

### PR DESCRIPTION
**Pull request by helrick**
CHEO-148

### CODE REVIEW

File: cre.sh [hunk 1]
```diff
@@ -148,11 +151,15 @@ function f_make_report
...
-	export denovo=1
+			export denovo=1
     fi
 
-    cre.gemini2txt.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variants.txt
-    cre.gemini.variant_impacts.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variant_impacts.txt
+    cre.gemini2txt.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variants.all.txt
+    cre.gemini.variant_impacts.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variant_impacts.all.txt
+
+    # remove duplicate lines from results of gemini query
```
Comment (dennis-kao) [inline cre.sh:160]: Why are we seeing duplicate lines from the gemini query? Should a unique clause be added to the query itself?

Comment (helrick) [reply cre.sh:160]: two queries, one for clinvar and one for regular so even if unique is put on both of them, duplicates will be present

File: cre.vcf2db.R [hunk 2]
```diff
@@ -41,7 +46,7 @@ genotype2zygocity <- function (genotype_str, ref){
 }
 
 # output : family.ensemble.txt
-create_report <- function(family, samples){
+create_report <- function(family, samples,type){
```
Comment (dennis-kao) [inline cre.vcf2db.R:49]: type is an unused variable

Comment (helrick) [reply cre.vcf2db.R]: Yep, good point! resolved in fd8bb17

**Pull request merged**